ikisocket.go

```diff
@@ -78,7 +79,7 @@ type EventPayload struct {
 	// Unique connection UUID
 	SocketUUID string
 	// Optional websocket attributes
-	SocketAttributes map[string]string
+	SocketAttributes map[string]interface{}
```

Comment:
I would prefer to leave it as string since if you use interface you're forced to cast every result

Reply:
True. What do you think of adding convenience methods to get certain types? Something like

```go
func (kws *Websocket) GetAttribute(key string) interface{}
func (kws *Websocket) GetStringAttribute(key string) string
func (kws *Websocket) GetIntAttribute(key string) int
...
func (kws *Websocket) SetAttribute(key string, value interface{})
```

Reply:
Perfect approach, I think  int, string, inferface no more types supported. Maybe when GO will support generic we can rewrite this with generics

Reply:
Great, I will add it. I don't think that GO will ever get generics, though.

Reply:
Done